src/video/wayland/SDL_waylandevents.c

```diff
@@ -1679,6 +1681,30 @@ static void data_device_handle_leave(void *data, struct wl_data_device *wl_data_
 static void data_device_handle_motion(void *data, struct wl_data_device *wl_data_device,
                                       uint32_t time, wl_fixed_t x, wl_fixed_t y)
 {
+    SDL_WaylandDataDevice *data_device = data;
+
+    if (data_device->drag_offer != NULL) {
+        /* TODO: SDL Support more mime types */
+        size_t length;
+        void *buffer = Wayland_data_offer_receive(data_device->drag_offer,
+                                                  &length, FILE_MIME, SDL_TRUE);
+        if (buffer) {
+            char *saveptr = NULL;
+            char *token = SDL_strtokr((char *)buffer, "\r\n", &saveptr);
+            while (token != NULL) {
+                char *fn = Wayland_URIToLocal(token);
+                if (fn) {
+                double dx;
```

Comment:
spacing looks wrong here?

Reply:
yes, not sure what happens. but that happen a few time with the recent :vi magic line removed ..
fixed

Reply:
yes I did install it !
maybe just a mistake